src/Orleans.CodeGenerator.MSBuild/Program.cs

```diff
@@ -52,7 +53,7 @@ public static int Main(string[] args)
                                     Console.WriteLine("Waiting for debugger to attach.");
                                 }
 
-                                Thread.Sleep(100);
+                                Task.Delay(100);
```

Comment:
`Task.Delay` must be awaited for it to have the desired effect.

Reply:
Thanks reuben , i revert commit